scripts/release/create_changelog.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -E
+set -o pipefail
+
+RELEASE_VERSION=$1
+PREVIOUS_RELEASE=$2
+
+REPOSITORY=${REPOSITORY:-kyma-project/template-operator}
+GITHUB_URL=https://api.github.com/repos/${REPOSITORY}
+GITHUB_AUTH_HEADER="Authorization: token ${GITHUB_TOKEN}"
+CHANGELOG_FILE="CHANGELOG.md"
+
+if [ "${PREVIOUS_RELEASE}"  == "" ]
+then
+  PREVIOUS_RELEASE=$(git describe --tags --abbrev=0)
```

Comment:
This assumes we create no other tags than release tags. Maybe looking at the existing releases and taking the most recent one is a better option?

Reply:
True, I will have a look if it's easily doable.

Reply:
Guess for the template-operator we only will have release tags. But also this is only executed if there is no PREVIOUS_RELEASE

Reply:
PREVIOUS_RELEASE is always empty, because that's how it's invoked in the `create-release.yml` workflow :thinking:

Reply:
No, this has been fixed by also providing `github.event.inputs.since`.